src/design-guidelines/color-system/examples/contrast-color-palette.scss

```diff
@@ -1,12 +1,6 @@
 @use '../../../style/functions';
```

Comment:
Colours for linear-progress are not working:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212056703-c6db5bde-d624-43b1-98fb-6b8d2aa7cf9b.png">

Reply:
⚡